digitalocean/hacktoberfest
# Require PRs be in a repo with hacktoberfest topic and be accepted

# Description

To reduce spam and make Hacktoberfest an opt-in event, only consider pull requests that are submitted in a repository that has 'hacktoberfest' as a repository topic. Further, only consider merged, approved or labelled (as 'hacktoberfest-accepted') PRs.

This does not apply to existing PRs.

Closes #583. 

# Test process

- Create PR in non-hacktoberfest-topic repo, PR should go to topic_missing state
- Create PR in hacktoberfest-topic repo, not approved, PR should go to not_accepted state
- Create PR in hacktoberfest-topic repo, merge or label as 'hacktoberfest-accepted' or approve, PR should go to waiting stae

# Requirements to merge
